doc/courses/rustconf/presentation/slides.md

```diff
@@ -119,7 +119,7 @@ Maybe...
 
   * `make` (just instrumenting `cargo`)
 
-  * Rust (nightly for `asm!`, compiling `core`, etc)
+  * Rust (nightly for `llvm_asm!`, compiling `core`, etc)
```

Comment:
haha, I realize this was probably the result of an automated change, but lets leave the old presentations as they were :)

Reply:
Oh, sorry about that :)) .  There shouldn't be any other left.